examples/isomorphic/run.release.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Do we need this file or can we just `cargo run -p isomorphic-server --target x86_64-unknown-linux-musl` ?

Reply:
Also it seems like this file isn't being used

Reply:
It's used by hand, I added it to the readme now.
I usually prefer executable examples to ones you have to copy-paste.